core/domain/role_services.py

```diff
@@ -108,8 +108,19 @@
 }
 
 
+def get_human_readable_role(role):
+    """converts role to human readable format.
+
+    Args:
+        role: str. The role to convert.
+    Returns:
+        str. Role in human readable format.
+    """
+    return role.lower().replace('_', ' ')
+
+
 def get_all_actions(role):
-    """Returns a list of all actions (including inherited actions)
+    """Returns a list of all actions(including inherited actions)
```

Comment:
oh, I got confused whether we use space before opening bracket in doc strings or not. Reverted back.